tests/test_expect.py

```diff
@@ -433,7 +433,7 @@ def test_before_after_exact(self):
         self._before_after(p)
 
     def _ordering(self, p):
-        p.timeout = 5
+        p.timeout = 20
```

Comment:
Well, I increased this because there was a timeout on one of the bots.

I think 5 seconds is too short, you will get timeouts now and then if the machine happens to be busy when running the test.